### PR TITLE
US 1583733 Add missing language IDs - 32

### PR DESCRIPTION
**Pull request by v-thpra**
US 1583733 - Fix CATS P1 issues by adding missing language IDs to code blocks.
- Used "console" for sample program console commands and console output examples.
- Used "output" for non-console output examples.
- Used "http" for line from web page example.

Contributes to #2192

### CODE REVIEW

**Review by HongGit — APPROVED** [review]

**Review by rpetrusha — DISMISSED** [review]
Thanks, @v-thpra. I've suggested some changes, one of which is unrelated to your PR. And thanks for creating the table -- it's a vast improvement!

File: docs/framework/wcf/samples/interoperating-with-asmx-web-services.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ Console.ReadLine();
   
  When you run the sample, the operation requests and responses are displayed in the client console window. Press ENTER in the client window to shut down the client.  
   
-```  
+```output
```
Comment (mairaw) [inline docs/framework/wcf/samples/interoperating-with-asmx-web-services.md]:
```suggestion
```console
```

**Review by mairaw — COMMENTED** [review]
A few comments

File: docs/framework/wcf/samples/extending-control-over-error-handling-and-reporting.md [hunk 2]
```diff
@@ -128,7 +128,7 @@ Press <ENTER> to terminate client.
   
  The file c:\logs\errors.txt contains the information logged about the errors by the service. Note that for the service to write to the directory you must make sure that the process under which the service is running, (typically ASP.NET or Network Service), has the permission to write to the directory.  
   
-```  
+```output
```
[the remark this answers is not in the document]
Comment (v-thpra) [reply docs/framework/wcf/samples/extending-control-over-error-handling-and-reporting.md]: That makes sense and it's more specific than "output".

**Review by mairaw — APPROVED** [review]

**Pull request merged**